modular_skyrat/modules/colony_fabricator/code/clothing/storage.dm

```diff
@@ -0,0 +1,88 @@
+// Backpacks
+
+/obj/item/storage/backpack/industrial/frontier_colonist
+	name = "frontier backpack"
+	desc = "A rugged backpack often used by settlers and explorers. Holds all of your equipment and then some."
+	icon = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing.dmi'
+	icon_state = "backpack"
+	worn_icon = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing_worn.dmi'
+	supports_variations_flags = CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON
+	worn_icon_teshari = 'modular_skyrat/modules/colony_fabricator/icons/clothes/clothing_worn_teshari.dmi'
+	worn_icon_state = "backpack"
+	inhand_icon_state = null
```

Comment:
It'd be nice if they had inhand icon states, that just feels like these items are incomplete, compared to every other items in the game trying their hardest to have at least some form of visual representation on the character when inhand.